ground/src/main/java/com/google/android/ground/persistence/remote/firebase/protobuf/ModelToProtoExt.kt

```diff
@@ -137,11 +136,7 @@ private fun toTaskData(id: String, newTaskData: TaskData) = taskData {
     is NumberTaskData -> numberResponse = numberResponse { number = newTaskData.value }
     // TODO: Ensure the dates are always converted to UTC time zone.
```

Comment:
Let's remove this TODO as well since the value is already in time since epoch in UTC

Reply:
Done